internal/extension/extension.go

```diff
@@ -14,6 +14,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"io"
 	"net/http"
```

Comment:
There are 4 places in this file where we call `Client.Do`. We should be sure to fix all of them. Esp start/end invocation and flush.

Reply:
Flush is a local test thing only, I don't actually think it works anymore